.travis.yml

```diff
@@ -4,15 +4,11 @@ rvm:
   - 2.3.1
 env:
```

Comment:
Do we have to do anything with the admin screens? Is this just to maintain the feature spec? Because we don't need it.